### PR TITLE
Fix cookbook : add field to a category

### PR DESCRIPTION
**Pull request by willy-ahva**
Two namespaces were wrong in the doc. It was CatalogBundle instead of AppBundle. Fixed it !

### CODE REVIEW

File: cookbook/category/add-properties-to-category.rst [hunk 1]
```diff
@@ -44,11 +44,11 @@ Define the Category Class
...
 You need to update your category entity parameter used in ``entities.yml`` file:
 
-.. literalinclude:: ../../src/Acme/Bundle/AppBundle/Resources/config/entities.yml
-   :language: yaml
-   :prepend: # /src/Acme/Bundle/AppBundle/Resources/config/entities.yml
-   :lines: 1-2
-   :linenos:
+.. code-block:: yaml
+
+    # /src/Acme/Bundle/AppBundle/Resources/entities.yml
```
Comment (fitn) [inline cookbook/category/add-properties-to-category.rst]: Resources/config/entities.yml

File: cookbook/category/add-properties-to-category.rst [hunk 2]
```diff
@@ -92,7 +92,10 @@ Then, don't forget to add your new field to the twig template:
 
 For the form validation you will have to add a new validation file:
 
-.. literalinclude:: ../../src/Acme/Bundle/CatalogBundle/Resources/config/validation/category.yml
-    :language: yaml
-    :prepend: # /src/Acme/Bundle/CatalogBundle/Resources/config/validation/category.yml
-    :linenos:
+.. code-block:: yaml
+
+    # /src/Acme/Bundle/AppBundle/Resources/validation/category.yml
```
Comment (fitn) [inline cookbook/category/add-properties-to-category.rst]: Resources/config/validation/category.yml

**Comment by fitn** [comment]
GTM once fixed

**Comment by nidup** [comment]
GTM

**Pull request merged**